mysql-test/include/plugin.defs

```diff
@@ -55,3 +55,4 @@ mysql_no_login     plugin/mysql_no_login      MYSQL_NO_LOGIN    mysql_no_login
 test_udf_services  plugin/udf_services TESTUDFSERVICES
 ha_tokudb          storage/tokudb             TOKUDB            tokudb,tokudb_trx,tokudb_locks,tokudb_lock_waits,tokudb_fractal_tree_info,tokudb_background_job_status
 tokudb_backup      plugin/tokudb_backup       TOKUDB_BACKUP     tokudb_backup
+ha_rocksdb         storage/rocksdb            ROCKSDB           rocksdb,rocksdb_cfstats,rocksdb_dbstats,rocksdb_perf_context,rocksdb_perf_context_global,rocksdb_cf_options,rocksdb_global_info,rocksdb_ddl,rocksdb_index_file_map
```

Comment:
A little of both. Actually is needed as the merged version from MYR-24 is missing the I_S plugin rocksdb_perf_context which is added here.